org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opent2t-translator-com-smartthings-binaryswitch",
-  "version": "1.0.0",
+  "version": "0.2.0",
```

Comment:
Current NPM packages for SmartThings are at 0.0.1, so updating to 0.2.0 for the new resource model should be fine. I propose that version is updated via a PR before publish to NPM so our GitHub main has the latest version at all times.